querybook/server/lib/notify/base_notifier.py

```diff
@@ -10,19 +12,37 @@ def notifier_name(self) -> str:
         """Name of the notifier that will be shown on the frontend"""
         raise NotImplementedError()
 
+    @property
+    @abstractmethod
```

Comment:
since this is a newly added method, should the behavior just be empty string or something like "This notifier does not provide custom notification"